### ISSUE TITLE
Temporary fix as website is down

### ISSUE THREAD

**Issue by yanarof** [VisualDataWeb/WebVOWL#202]
Since the online version of this website is down http://vowl.visualdataweb.org/webvowl.html
means that the Dockerfile is unable to download the `ROOT.war` file with `wget -O /usr/local/tomcat/webapps/ROOT.war http://vowl.visualdataweb.org/downloads/webvowl_1.1.7.war`

Until the website is back online, a temporary fix is to download this file from another source.
This is what the Dockerfile should look like in order to build correctly:
```Dockerfile
FROM tomcat:9-jre8-alpine
ARG version=1.1.7
RUN rm -rf /usr/local/tomcat/webapps/* && \
     wget -O /usr/local/tomcat/webapps/ROOT.war https://github.com/DrSnowbird/WebVOWL-docker/raw/master/webvowl_1.1.7.war
CMD ["catalina.sh", "run"]
```

I guess it was not included directly in this repository is because of its size of ~25Mb ?

**Comment by giovannidegani** [comment]
how about building the war file ?

**Comment by yanarof** [comment]
Yes, it would be even better, but I had no time to dig into that path (it was a quick & dirty fix)
Do you have a command that allows to build it ?
